lib/screens/log_screen_data.ex

```diff
@@ -33,29 +33,34 @@ defmodule Screens.LogScreenData do
     end
   end
 
-  def log_api_response(%{force_reload: true}, screen_id, last_refresh, is_screen) do
-    log_api_response_success(screen_id, last_refresh, is_screen)
+  def log_api_response(%{force_reload: true, status: status}, screen_id, last_refresh, is_screen) do
+    log_api_response_success(screen_id, last_refresh, is_screen, status)
   end
 
-  def log_api_response(%{success: true}, screen_id, last_refresh, is_screen) do
-    log_api_response_success(screen_id, last_refresh, is_screen)
+  def log_api_response(%{success: true, status: status}, screen_id, last_refresh, is_screen) do
+    log_api_response_success(screen_id, last_refresh, is_screen, status)
+  end
+
+  def log_api_response(status, screen_id, last_refresh, is_screen)
+      when is_atom(status) do
```

Comment:
Is there a specific use-case that requires us to have this `is_atom/1` check in place?

Reply:
Nope, that was something I needed in my first idea. Just forgot to delete it when I went another direction. Good catch.